bedrock/firefox/views.py

```diff
@@ -851,7 +834,7 @@ class FirefoxHubView(BlogPostsView):
     def get_template_names(self):
         locale = l10n_utils.get_locale(self.request)
 
-        if switch('firefox-57-release') and lang_file_is_active('firefox/hub/home-quantum', locale):
+        if lang_file_is_active('firefox/hub/home-quantum', locale):
```

Comment:
Should we do away with this whole "hub" nomenclature?

Reply:
Hmm, I don't think we can rename lang files without breaking things?

Reply:
Oh, sorry! I was talking about the bedrock directory structure/file naming, not really the lang files (I just had to pick spot in the file where GitHub would let me comment). Seems a little odd to have the Firefox home page in a `hub` directory.